lib/graphs/digraph.nit

```diff
@@ -0,0 +1,606 @@
+# Implementations for representing directed graphs, also called digraphs.
+#
+# Currently, there are two classes
```

Comment:
I know this is a WIP, but later on, could you add some precisions on the classes of the module or remove this comment since the classes document themselves ?